.github/workflows/design-system-component-testing.yml

```diff
@@ -9,6 +9,10 @@ on:
 jobs:
   cypress-run:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
```

Comment:
If tests fail on Chrome, tests on Firefox must continue, and the opposite is truthy too.